ekf_se_map:
  frequency: 10
  two_d_mode: true
  print_diagnostics: false
  publish_tf: true
  transform_time_offset: 0.0

  map_frame: map
  odom_frame: odom
  base_link_frame: base_link
  world_frame: odom

  odom0: odometry/gps
  odom0_config: [true,  true,  false,
                false, false, false,
                false, false, false,
                false, false, false,
                false, false, false]

  # imu0: /ML/Imu/Data
  # imu0_config: [false, false, false,
  #               false,  false,  true,
  #               false, false, false,
  #               false,  false,  false,
  #               false,  false,  false]
  # imu0_nodelay: false
  # imu0_differential: false
  # imu0_relative: false
  # imu0_queue_size: 5
  
  imu0: gps/navheading
  imu0_config: [false, false, false,
                false,  false,  true,
                false, false, false,
                false,  false,  false,
                false,  false,  false]
  imu0_nodelay: false
  imu0_differential: false
  imu0_relative: false
  imu0_queue_size: 5
  
